core/shared/src/main/scala/fs2/Segment.scala

```diff
@@ -835,6 +835,27 @@ abstract class Segment[+O,+R] { self =>
   }
 
   /**
+   * Converts this segment to an array, discarding the result.
+   *
```

Comment:
Ah sorry. I am getting this. It was just unclear to me from the toArray signature what is lost. Thanks @mpilquist .